packages/playground/blueprints/src/lib/compile.ts

```diff
@@ -237,14 +237,9 @@ export function compileBlueprint(
 						onStepCompleted(result, step);
 					} catch (e) {
 						console.error(e);
-						throw new Error(
-							`Error when executing the blueprint step #${i} (${JSON.stringify(
-								step
-							)}). Inspect the "cause" property of this error for more details`,
-							{
-								cause: e,
-							}
-						);
+						throw new Error(`Error when executing the blueprint step #${i} (${JSON.stringify(
+							step
+						)}) ${e instanceof Error ? `: ${e.message}` : e}`);
```

Comment:
Why remove the `cause`? AFAIR it connects the two stack traces together and you can keep on expanding in dev tools to get to the bottommost error.

Reply:
All messages need to be strings in the logger to support any environment. 
I can return the old logging here and use the string for the logger only.

Reply:
Removing the `Inspect the "cause" property of this error for more details` part sounds fine, I'd just keep the `{ cause: e }` argument to retain the error stack.

Reply:
Done in [ef4993b](https://github.com/WordPress/wordpress-playground/pull/1035/commits/ef4993bd967a47dfd80858a8e43f2f5cba6f8fef)